tw/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Elixir School
+redirect_from: /
```

Comment:
We need to remove these `redirect_from:` 👍

Reply:
Sure. I already known this issue from #1282 . I will remove all the `redirect_from` from my every translation so far. 👍